public/static/locales/isl/about.json

```diff
@@ -1,12 +1,12 @@
 {
     "why": {
         "title": "Af hverju?",
-        "beginning": "Á síðstu árum hefur verið bylting í raddtækni og því hvernig við notum röddina til þess að stjórna tækninni. Íslenskan á undir högg að sækja vegna þeirra öru tæknibreytinga en mörg okkar eiga nú þegar samskipti við tölvur og ýmis tæki á erlendu máli. Fólk mun nota röddina í auknum mæli til að stýra hvers kyns tækjum og tólum en vandinn er sá að tækin skilja ekki íslensku.",
+        "beginning": "Á síðstu árum hefur verið bylting í raddtækni og því hvernig við notum röddina til þess að stjórna tækninni. Íslenskan á undir högg að sækja vegna þessara öru tæknibreytinga en mörg okkar eiga nú þegar samskipti við tölvur og ýmis tæki á erlendu máli. Fólk mun nota röddina í auknum mæli til að stýra hvers kyns tækjum og tólum en vandinn er sá að tækin skilja ekki íslensku.",
```

Comment:
My Icelandic person says þeirra is more correct.

Reply:
Perhaps.
But we both missed this one: "en mörg okkar eiga" --> "en mörg okkar eigum"